templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+<ul class="sidebar-nav">
+  <li>
+    <a href="{% url 'taxi:index' %}">Home page</a>
+  </li>
+  <li>
+    <a href="#">Manufacturers</a>
+  </li>
+  <li>
+    <a href="#">Cars</a>
+  </li>
+  <li>
+    <a href="#">Drivers</a>
```

Comment:
The links for 'Manufacturers', 'Cars', and 'Drivers' are currently placeholders (`href="#"`). Once you have the appropriate URLs or views set up, replace these with `{% url 'app_name:view_name' %}` to ensure they point to the correct pages.